richtext/plugins/org.polarsys.kitalpha.richtext.nebula.widget/src/org/polarsys/kitalpha/richtext/nebula/widget/MDENebulaRichTextConfiguration.java

```diff
@@ -92,21 +92,28 @@ private void doAddItems(String toolbar, String... items) {
 		}
 	}
 
+	/*
+	 * This method has been modified by commit 6423c4.
```

Comment:
can you remove the commit id. (as when it will be merge to master branch, the commit id will be different)